fl4health/utils/losses.py

```diff
@@ -21,7 +21,13 @@ def __init__(
     def as_dict(self) -> Dict[str, float]:
         loss_dict: Dict[str, float] = {}
         loss_dict["checkpoint"] = float(self.checkpoint.item())
-        loss_dict["backward"] = float(self.backward.item())
+
```

Comment:
In the previous iteration, we had to do a float(...) conversion. Is that no longer necessary?